scripts-available/CDB_ForeignTable.sql

```diff
@@ -0,0 +1,122 @@
+---------------------------
+-- FDW MANAGEMENT FUNCTIONS
+--
+-- All the FDW settings are read from the `cdb_conf.fdws` entry json file.
+---------------------------
+
+CREATE OR REPLACE FUNCTION cartodb._CDB_Create_FDW(name text, config json)
```

Comment:
Right now there are three things which share the same name:
- the config entry name
- the schema name
- the `FOREIGN SERVER` name

I wanted to keep them the same as a convention, but you're right about `name` being too generic. What about `fdw_name`?